python/paddle/tensor/math.py

```diff
@@ -5051,6 +5052,57 @@ def lgamma_(x, name=None):
         return _C_ops.lgamma_(x)
 
 
+def multigammaln(x, p, name=None):
```

Comment:
code is fine, but The signature of API in [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20230913_api_design_for_multigammaln.md?plain=1#L152) needs to be consistent with the code, add default value parameter `name`

Reply:
done.
- https://github.com/PaddlePaddle/community/pull/750